code/_onclick/ventcrawl.dm

```diff
@@ -29,6 +29,33 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 		return C.check_clothing(src)
 	return 1
 
+/obj/item/clothing/under/contortionist/verb/crawl_through_vent()
+	set name = "Crawl Through Vent"
+	set category = "Object"
+	set src in usr
+
+	var/mob/living/carbon/human/user = usr
+	if(istype(user) && user.w_uniform == src && check_clothing(user))
+		var/pipe = user.start_ventcrawl()
+		if(pipe)
+			user.handle_ventcrawl(pipe)
+
+/mob/proc/start_ventcrawl()
+	var/atom/pipe
+	var/list/pipes = list()
+	for(var/obj/machinery/atmospherics/unary/U in range(1))
+		if(is_type_in_list(U,ventcrawl_machinery) && Adjacent(U))
```

Comment:
Should probably add a can_ventcrawl_into(U) here, in case checks need to be added in the future.